pom.xml

```diff
@@ -57,7 +57,7 @@
     <com.ibm.icu.version>56.1</com.ibm.icu.version>
     <com.lowagie.itext.version>2.1.7</com.lowagie.itext.version>
     <com.mattbertolini.version>2.0.0</com.mattbertolini.version>
-    <com.smartgwt.version>4.0</com.smartgwt.version>
+    <com.smartgwt.version>6.0</com.smartgwt.version>
```

Comment:
As explained in [DEP-153](https://jira.exoplatform.org/browse/DEP-153) , the Maven coordinates have changed.
Please update this dependency with 
```
<groupId>com.isomorphic.smartgwt.lgpl</groupId>
<artifactId>smartgwt-lgpl</artifactId>
<version>6.0-p20170514</version>
```

So do not forget to:

1. rename the Maven property to `com.isomorphic.smartgwt.lgpl.version`
2. update the coordinates with the new version / groupId / artifactId
3. re-order both maven property and maven dependency declaration to keep the alphabetical order

thanks